src/DRICH_geo.cpp

```diff
@@ -40,7 +40,7 @@ static Ref_t createDetector(Detector& desc, xml::Handle_t handle, SensitiveDetec
   xml::Component        dims    = detElem.dimensions();
   OpticalSurfaceManager surfMgr = desc.surfaceManager();
   DetElement            det(detName, detID);
-  sens.setType("tracker");
+  sens.setType("opticaltracker");
```

Comment:
The way we should roll this out is probably by keeping the original tracker type and setting the SD action in npsim.py. Otherwise we break ddsim functionality for all released versions.

Reply:
Yes, I will try again later using `tracker`. I didn't notice the new `npsim` until after adding `opticaltracker` to DD4hep; and after doing this, it's clear that wasn't the best way.

Reply:
reverted to `tracker`